docs/Configuration.md

```diff
@@ -4,18 +4,97 @@ title: Configuration
 permalink: /configuration/
 nav_order: 4
 ---
-# Configuration 
+
+# Configuration
 
 VS Code settings enable you to customize various aspects of its function. The InterSystems extensions provide settings used to configure VS Code for ObjectScript development.
 
+{: #code-configuration-basic}
+
+## Basic Configuration
+
+VS Code has a concept of a workspace, which is a set of directories you want to use when you're working on a particular project. In the simplest setup when you are working within a single directory, a VS Code workspace is just the root folder of your project. In this case you keep your settings in two files inside a `.vscode` directory located at the root of your project. Those two files are `settings.json`, which contains most configuration settings, and `launch.json`, which contains debugging configurations.
+
+Here is the simplest `settings.json` file contents:
+
+{: #code-workspace-simple}
+
+```json
+{
+    "objectscript.conn": {
+        "server": "iris",
+        "ns": "USER",
+        "active": true, 
+        "host": "localhost", 
+        "port": 52773, 
+        "username": "_SYSTEM" 
+    }
+}
+```
+
+If you need ObjectScript compilation flags other than the default, add an `"objectscript.compileFlags"` property to `settings.json` (more compileFlags options are [covered here](/vscode-objectscript/settings/):
+
+{: #code-workspace-compileFlags}
+
+```json
+{
+    "objectscript.conn": {
+        "server": "iris",
+        "ns": "USER",
+        "active": true, 
+        "host": "localhost", 
+        "port": 52773, 
+        "username": "_SYSTEM" 
+    },
+    "objectscript.compileFlags": "cuk/compileembedded=1"
+}
+```
+
+Here is the simplest `launch.json` file contents, which debugs the method `Test` in the class `Example.Service`, passing 2 parameters as input:
```

Comment:
This is all covered already on the run/debug page.

Reply:
Same as above. Some duplication is good in documentation. This being the first section people read, it's valuable to get them up and running as fast as possible. I'll add a reference to the more comprehensive page.

Reply:
Added a reference to the debugging section